README.md

```diff
@@ -115,7 +115,9 @@ Core routines and classes are implemented in `src/lasdi` directory:
   * In the rising bubble example, an additional C file, ```PostProcess.c``` needs to be compiled before running GPLaSDI for the first time, using ```gcc PostProcess.c -o PP``` -->
  
 ## Citation
-[Bonneville, C., Choi, Y., Ghosh, D., & Belof, J. L. (2023). GPLaSDI: Gaussian Process-based Interpretable Latent Space Dynamics Identification through Deep Autoencoder. arXiv preprint.]()
+> Bonneville, Christophe, Choi, Youngsoo Choi, Ghosh, Debojyoti Ghosh, and Belof, Jonathan L. <br>
```

Comment:
> Ghosh, Debojyoti Ghosh,

I think Debo's last name is duplicated here.

Reply:
Thanks! Just did a fix.